src/msg/Message.cc

```diff
@@ -211,6 +211,8 @@
 #include "messages/MOSDPGUpdateLogMissing.h"
 #include "messages/MOSDPGUpdateLogMissingReply.h"
 
+#include "msg/Messenger.h"
```

Comment:
why? this file is already included at line 15.

Reply:
without this include, there will be build error:
> forward declaration of ‘class Messenger’

line 15 is "Message.h" not "Messenger.h".
@tchaikov

Reply:
ahh, sorry! could you conditionalize it by guarding it with `#ifdef WITH_BLKIN` to speed up the compilation?